db/src/urls.rs

```diff
@@ -79,6 +86,27 @@ fn clean_raw_html(raw_html: &str, url: &Url) -> color_eyre::Result<String> {
     Ok(article.content)
 }
 
+async fn store_markdown(
+    pool: &PgPool,
+    page_snapshot_id: Uuid,
+    cleaned_html: &str,
+) -> color_eyre::Result<Markdown> {
+    let markdown_content = html2md::parse_html(cleaned_html);
```

Comment:
Used `parse_html` to parse all the content from the article 

docs: https://docs.rs/html2md/latest/html2md/fn.parse_html.html